tools/tpm2_sign.c

```diff
@@ -289,8 +290,8 @@ static bool init(int argc, char *argv[], tpm_sign_ctx *ctx) {
         }
     }
 
-    if (!((flags.k || flags.c) && flags.g && flags.m && flags.s)) {
-        LOG_ERR("Expected options (k or c) and g and m and s");
+    if (!((flags.k || flags.c) && flags.m && flags.s)) {
+        LOG_ERR("Expected options (k or c) and m and s");
         return false;
```

Comment:
This is an unrelated fix (-g option isn't needed because sha1 is the default), so I think it should be in a separate patch with its own explanation why is needed.